test/extended/operators/images.go

```diff
@@ -20,6 +21,15 @@ import (
 
 var _ = Describe("[sig-arch] Managed cluster", func() {
 	oc := exutil.NewCLIWithoutNamespace("operators")
+
+	BeforeEach(func() {
+		microShift, err := exutil.IsMicroShiftDeployment(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if microShift {
+			Skip("Skipped in MicroShift")
```

Comment:
Why is this test no applicable for MicroShift?

Reply:
Realized I could simplify it. The test uses `oc adm release info` which requires access to `config.openshift.io`, so its tagged now.